index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="">
```

Comment:
The `lang` attribute in the `<html>` tag is empty. It's recommended to specify a language code, such as `en` for English, to improve accessibility and SEO.